packages/apps/workshop/stories/design-system/components/action-menu/action-menu-webcomponent.stories.js

```diff
@@ -15,6 +15,7 @@ export default {
   title: 'Design System/Components/Action Menu/WebComponent',
 
   parameters: {
+    docs: { iframeHeight: 200 },
```

Comment:
Not possible to use common/base config and override it if needed ?

Reply:
Yep it is possible, I will define `{ iframeHeight: 200 }` as the default

Reply:
Hmm in fact there's already a default iframeHeight which is set to `60px`. (See https://github.com/storybookjs/storybook/tree/next/addons/docs/common#iframe-height). We could keep it as is, as in one way or another some stories will need some override of this setting